import_flow_events.py

```diff
@@ -84,18 +130,81 @@
     CREDENTIALS 'aws_access_key_id={aws_access_key_id};aws_secret_access_key={aws_secret_access_key}'
     FORMAT AS CSV;
 """
-
-Q_CHECK_FOR_DAY = """
-    SELECT timestamp FROM flow_events
-    WHERE timestamp >= {timestamp}
-    AND timestamp < {timestamp} + 86400
-    LIMIT 1;
+Q_INSERT_METADATA = """
+    WITH durations AS (
+      SELECT flowId, MAX(flowTime)
+      FROM flow_csv
+      GROUP BY flowId
```

Comment:
Haven't actually tried this query out in isolation yet, maybe it doesn't do what I think it does.